.github/CODEOWNERS

```diff
@@ -0,0 +1,21 @@
+# CODEOWNERS adds reviewers depending on the location of the change.
```

Comment:
As a note, this is apparently branch specific. If you'd like more info, here's some documentation on codeowners:
https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners